static/icons.svg

```diff
@@ -48,4 +48,5 @@
 <symbol id="Ellipsis" xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 20 18" ><circle opacity="0.6" cx="2" cy="11" r="2" fill="currentColor"/><circle opacity="0.6" cx="10" cy="11" r="2" fill="currentColor"/><circle opacity="0.6" cx="18" cy="11" r="2" fill="currentColor"/></symbol>
 <symbol id="Checked" xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 19 18"><path d="M15.6875 5.0625L7.8125 12.9375L3.875 9" stroke="currentColor" stroke-width="1.5" stroke-linecap="round" stroke-linejoin="round"/></symbol>
 <symbol id="ArrowElbowDownRight" xmlns="http://www.w3.org/2000/svg" width="24" height="24" fill="currentColor" viewBox="0 0 256 256"><rect width="256" height="256" fill="none"></rect><polyline points="160 128 208 176 160 224" fill="none" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="16"></polyline><polyline points="64 32 64 176 208 176" fill="none" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="16"></polyline></symbol>
+<symbol id="Info" xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none"><path fill-rule="evenodd" clip-rule="evenodd" d="M12 3.75C7.44365 3.75 3.75 7.44365 3.75 12C3.75 16.5563 7.44365 20.25 12 20.25C16.5563 20.25 20.25 16.5563 20.25 12C20.25 7.44365 16.5563 3.75 12 3.75ZM2.25 12C2.25 6.61522 6.61522 2.25 12 2.25C17.3848 2.25 21.75 6.61522 21.75 12C21.75 17.3848 17.3848 21.75 12 21.75C6.61522 21.75 2.25 17.3848 2.25 12ZM10.5 11.25C10.5 10.8358 10.8358 10.5 11.25 10.5H12C12.4142 10.5 12.75 10.8358 12.75 11.25V15.75C13.1642 15.75 13.5 16.0858 13.5 16.5C13.5 16.9142 13.1642 17.25 12.75 17.25H12C11.5858 17.25 11.25 16.9142 11.25 16.5V12C10.8358 12 10.5 11.6642 10.5 11.25ZM12.9375 7.875C12.9375 8.49632 12.4338 9 11.8125 9C11.1912 9 10.6875 8.49632 10.6875 7.875C10.6875 7.25368 11.1912 6.75 11.8125 6.75C12.4338 6.75 12.9375 7.25368 12.9375 7.875Z" fill="currentColor"/></symbol>
```

Comment:
Is this icon going to be used in a future task?

Reply:
It is going to be used in the icon options in the CMS. 
<img width="331" alt="image" src="https://user-images.githubusercontent.com/10616243/167723593-f95820b4-0b75-4f7a-9299-2080b8bd7a1f.png">

Reply:
I see! 

Then let me tell you about [svgomg](https://jakearchibald.github.io/svgomg/). It's a service that can optimize SVGs by removing some unnecessary markup. Running this new icon generated a ~55% reduction in size. You just need to replace `symbol` for `svg` when pasting the markup there.
![CleanShot 2022-05-10 at 20 47 26](https://user-images.githubusercontent.com/381395/167742486-35b8bab9-7aae-4812-93aa-b0ba21f19df7.png)

```suggestion
<symbol id="Info" xmlns="http://www.w3.org/2000/svg" width="24" height="24" fill="none"><path fill-rule="evenodd" clip-rule="evenodd" d="M12 3.75a8.25 8.25 0 1 0 0 16.5 8.25 8.25 0 0 0 0-16.5ZM2.25 12c0-5.385 4.365-9.75 9.75-9.75s9.75 4.365 9.75 9.75-4.365 9.75-9.75 9.75S2.25 17.385 2.25 12Zm8.25-.75a.75.75 0 0 1 .75-.75H12a.75.75 0 0 1 .75.75v4.5a.75.75 0 0 1 0 1.5H12a.75.75 0 0 1-.75-.75V12a.75.75 0 0 1-.75-.75Zm2.438-3.375a1.125 1.125 0 1 1-2.25 0 1.125 1.125 0 0 1 2.25 0Z" fill="currentColor"/></symbol>
```

Reply:
That's gold, @filipewl ! Just create [this Jira task](https://vtex-dev.atlassian.net/browse/FSSS-337), so we can do the same with other icons.